ios/Video/NewPlayerView.swift

```diff
@@ -0,0 +1,394 @@
+                uri: sourceValue.uri,
+                drm: rndvJSDrm,
+                progressUpdateInterval: sourceValue.progressUpdateInterval ?? 6,
+                type: sourceValue.type,
+                title: sourceValue.title ?? "",
+                live: sourceValue.live,
+                partialVideoInformation: jsPartialVideoInformation,
+                isAudioOnly: sourceValue.isAudioOnly,
+                config: jsConfig,
+                imageUri: sourceValue.imageUri,
+                thumbnailsPreview: sourceValue.thumbnailsPreview,
+                resumePosition: nil,
+                delay: nil,
+                ads: jsAds,
+                subtitles: jsSubtitles,
+                limitedSeekableRange: jsLimitedSeekableRange,
+                selectedAudioTrack: nil,
+                selectedSubtitleTrack: sourceValue.selectedSubtitleTrack,
+                selectedPlaybackQuality: nil,
+                nowPlaying: jsNowPlaying)
+        }
+        
+        var jsTranslations: RNDReactNativeDiceVideo.JSTranslations?
+        if let translationsValue = self.jsProps.translations.value {
+            var dorisTranslationsViewModel = DorisTranslationsViewModel()
+            dorisTranslationsViewModel.play = translationsValue.playerPlayButton
+            dorisTranslationsViewModel.pause = translationsValue.playerPauseButton
+            dorisTranslationsViewModel.stats = translationsValue.playerStatsButton
+            dorisTranslationsViewModel.audioAndSubtitles = translationsValue.playerAudioAndSubtitlesButton
+            dorisTranslationsViewModel.live = translationsValue.live
+//            dorisTranslationsViewModel.goLive = translationsValue.goLive
+            dorisTranslationsViewModel.favourites = translationsValue.favourite
+//            dorisTranslationsViewModel.watchlist = translationsValue.watchlist
+            dorisTranslationsViewModel.moreVideos = translationsValue.moreVideos
+//            dorisTranslationsViewModel.captions = translationsValue.captions
+            dorisTranslationsViewModel.rewind = translationsValue.rewind
+            dorisTranslationsViewModel.fastForward = translationsValue.fastForward
+            dorisTranslationsViewModel.audio = translationsValue.audioTracks
+            dorisTranslationsViewModel.info = translationsValue.info
+            dorisTranslationsViewModel.adsCountdownAd = translationsValue.adsCountdownAd
+            dorisTranslationsViewModel.adsCountdownOf = translationsValue.adsCountdownOf
+            dorisTranslationsViewModel.annotations = translationsValue.annotations
+            dorisTranslationsViewModel.playingLive = translationsValue.playingLive
+            dorisTranslationsViewModel.nowPlaying = translationsValue.nowPlaying
+//            dorisTranslationsViewModel.tvPlayerEPG = translationsValue.tvPlayerEPG
+            jsTranslations = RNDReactNativeDiceVideo.JSTranslations(beaconTranslations: nil, dorisTranslations: dorisTranslationsViewModel)
```

Comment:
what are those commented ones?

Reply:
need confirm Translations have those value, but JSTranslations not have